airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -0,0 +1,170 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import re
+from contextlib import closing
+from copy import copy
+from typing import Any, Dict, List, Optional, Union
+
+from databricks import sql
+from databricks.sql.client import Connection
+
+from airflow.exceptions import AirflowException
+from airflow.hooks.dbapi import DbApiHook
+from airflow.providers.databricks.hooks.databricks_base import DatabricksBaseHook
+
+
+class DatabricksSqlHook(DatabricksBaseHook, DbApiHook):
+    """
+    Interact with Databricks SQL.
+
+    :param databricks_conn_id: Reference to the :ref:`Databricks connection <howto/connection:databricks>`.
+    :param http_path: Optional string specifying HTTP path of Databricks SQL Endpoint or cluster.
+        If not specified, it should be specified in the Databricks connection's extra parameters.
+    :param session_configuration: An optional dictionary of Spark session parameters. Defaults to None.
+        If not specified, it could be specified in the Databricks connection's extra parameters.
+    """
+
+    conn_name_attr = 'databricks_conn_id'
```

Comment:
These 3 fields are already present  **databricks_base**,
I don't think we should have to redefine them here.